keyboards/handwired/10k/info.json

```diff
@@ -0,0 +1,30 @@
+{
+  "keyboard_name": "10k",
+  "maintainer": "Nabos",
+  "url": "",
+  "matrix_pins": {
+    "cols": ["C6", "D7", "E6", "B4", "B5"],
+    "rows": ["B6"]
+  },
+  "usb": {
+    "device_ver": "0x0001",
```

Comment:
```suggestion
    "device_version": "0.0.1",
```

Reply:
I'm not sure about this suggestion.
When it try it, I get this error:
`usb: Additional properties are not allowed ('device_version' was unexpected)`
And I cannot find it in other keyboards, all I find is things like this:
https://github.com/qmk/qmk_firmware/blob/9dea6f772077dc5c09daf40378e45884d29ab2e2/keyboards/clueboard/60/info.json#L29
But in the documentation it clearly noted "device_version":
https://github.com/qmk/qmk_firmware/blob/master/docs/reference_info_json.md#usb

What do you suggest ?

Reply:
![image](https://user-images.githubusercontent.com/4781841/157328918-87ad25af-f5ec-4129-86ac-19165378c2e7.png)

Reply:
I'm truly sorry. I don't know how I could miss that.